src/libs/ReportUtils.js

```diff
@@ -510,6 +510,16 @@ function navigateToDetailsPage(report) {
     Navigation.navigate(ROUTES.getReportParticipantsRoute(report.reportID));
 }
 
+/**
+ * Generate a random reportID between 98000000 (the number of reports before the switch from sequential to random)
+ * and the maximum safe integer of js (53 bits aka 9,007,199,254,740,991)
+ *
+ * @returns {number}
+ */
+function generateRandomReportID() {
```

Comment:
NAB, but was thinking that the "randomness" is a feature of a generated reportID, but not necessarily something we need to think about when generating one - so this could maybe be simplified to `generateReportID()`.

Reply:
Sounds good to me!